kde-misc/playbar2/playbar2-2.5.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+DESCRIPTION="MPRIS2 client, written in QML for Plasma 5"
+HOMEPAGE="https://github.com/audoban/PlayBar2"
+SRC_URI="https://github.com/audoban/PlayBar2/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE=""
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="$(add_frameworks_dep extra-cmake-modules)
+	$(add_frameworks_dep kconfigwidgets)
+	$(add_frameworks_dep kdeclarative)
+	$(add_frameworks_dep kdoctools)
+	$(add_frameworks_dep kglobalaccel)
+	$(add_frameworks_dep kwindowsystem)
+	$(add_frameworks_dep kxmlgui)
+	$(add_frameworks_dep plasma)
+	$(add_plasma_dep plasma-workspace)
```

Comment:
There is no linking to `kde-plasma/plasma-workspace`, why is it here?

Reply:
Upstream gives that as a dependency in the README.md. Should I simply remove it?

Reply:
removed this, because it is not referenced in cmake